workloadDriver/src/main/java/com/vmware/weathervane/workloadDriver/common/core/loadPath/FindMaxLoadPath.java

```diff
@@ -232,6 +229,11 @@ private UniformLoadInterval getNextInitialRampInterval() {
 		curStatusInterval.setStartUsers(curUsers);
 		curStatusInterval.setEndUsers(curUsers);
 		curStatusInterval.setDuration(nextIntervalDuration);
+		
+		statsIntervalComplete = true;
+		curStatsInterval.setName(nextInterval.getName());
+		curStatsInterval.setUsers(curUsers);
+		curStatsInterval.setDuration(nextIntervalDuration);
```

Comment:
this line were just set above. do they need to be set again?

Reply:
The ones above are setting the curStatusInterval, while these are setting the curStatsInterval. These  are used for different purposes.  It is possible that we don't need both now that I am handling the statsIntervals this way, but I don't want to break something trying to figure it out. At some point I would like to clean up the loadPath code, but not now.